ui/app/components/ui/toast.tsx

```diff
@@ -28,6 +28,8 @@ const toastVariants = cva(
     variants: {
       variant: {
         default: "border bg-background text-foreground",
+        // TODO: Implement success variant styles
+        success: "border bg-background text-foreground",
```

Comment:
Didn't want to change existing styles arbitrarily but I think it's worth having a separate variant here.